matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/SendEventWorker.kt

```diff
@@ -47,12 +47,10 @@ internal class SendEventWorker(context: Context,
     @JsonClass(generateAdapter = true)
     internal data class Params(
             override val sessionId: String,
-            // TODO remove after some time, it's used for compat
             val event: Event? = null,
             val eventId: String? = null,
             val roomId: String? = null,
             val type: String? = null,
-            val contentStr: String? = null,
```

Comment:
Maybe keep it for the moment during the migration?

Reply:
It won't crash as it's not mandatory, so I think it's OK